reactioncommerce/reaction
# Convert TaxCloud panel to React

Resolves #3093 .

This PR converts the TaxCloud settings panel from using [AutoForm](https://github.com/aldeed/meteor-autoform) to using [React](https://reactjs.org/) for editing its settings.

## Test Instructions
- Login as admin.
- Open the TaxCloud settings panel. Observe the current API Login ID and API Key values (if any).
- Change the login ID or/and API key in the TaxCloud settings form and submit the form. Observe an alert that confirms that the new settings were saved successfully.
- Refresh the page and open the TaxCloud panel again. Observe that the form now contains the login ID or API key you entered above.
- Note: Instead of refreshing the page, you can use RoboMongo (or similar) to inspect the DB. Just search the `Package` collection for the package with `name: taxes-taxcloud`. Then cycle between editing the TaxCloud settings form and checking the `settings.taxcloud` field in the said package. You'll notice that that field always changes as appropriate.
